.github/actions/setup-deploy/action.yaml

```
@@ -78,7 +78,7 @@ runs:
 
     # Pin kubectl version to 1.23 otherwise interactions with k8s clusters versioned <=1.21 won't work.
     # See https://github.com/2i2c-org/infrastructure/issues/1271.
-    - uses: azure/setup-kubectl@v3.1
+    - uses: azure/setup-kubectl@v3.2
```

:+1: for pinning this to a major version to reduce PRs like this and count on them to not break things without bumping the major version.

```suggestion
    - uses: azure/setup-kubectl@v3
```

---

The 3.2 release was "Uses the new kubectl download uri" and their 3.1 release was "Bump @actions/core to prevent warning output."

It seems fine to go with `v3`.